### PR TITLE
Curve USDT Peg Test

### PR DESCRIPTION
**Pull request by raddy**
Super simple Curve based USDT peg test

### CODE REVIEW

File: contracts/tether/Ante_USDTCurvePegTest.sol [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     * USDC Contract: https://etherscan.io/token/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
+     */
+    function checkTestPasses() public view override returns (bool) {
+        (, uint256 price) = ICurveFiSwap(CurveFiRegistry).get_best_rate(
+            TetherAddr,
+            0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48,
+            100000000000
+        );
+        console.log("Will receive %s tokens", price);
+        return (90000000000 < price);
```
Comment (a-deva) [inline contracts/tether/Ante_USDTCurvePegTest.sol:39]: Is this attackable using a flash loan? If so, using a TWAP oracle or preventing flashloan attacks by forcing the msg.sender to be tx.origin is necessary

**Review by a-deva — COMMENTED** [review]
Thanks raddy!

File: contracts/tether/Ante_USDTCurvePegTest.sol (hunk 1 above)
Comment (raddy) [reply contracts/tether/Ante_USDTCurvePegTest.sol:39]: ah very true; I think I want to add a better failure case test for this too

**Comment by delalunia** [comment]
Deprecated, merged with https://github.com/antefinance/ante-community-tests/pull/19

**Pull request closed without merging**